addons/sms/models/mail_thread.py

```diff
@@ -305,13 +305,14 @@ def _notify_thread_by_sms(self, message, recipients_data, msg_vals=False,
                 value['sanitized'] or original
                 for original, value in sanitized.items()
             ]
+            existing_partners_numbers = [vals_dict['number'] for vals_dict in sms_create_vals]
```

Comment:
as an alternative we could use a set, but if there is not too many number it's not too much of an improvement

```suggestion
            existing_partners_numbers = {vals_dict['number'] for vals_dict in sms_create_vals}
```

Reply:
Appreciate good tip! Updating list into a set.